roles/journalbeat/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+journalbeat_image: tpolekhin/journalbeat
```

Comment:
is this a custom image? if so, is there a github repo and can it be pushed to the ciscocloud organization instead?